scripts/superdesk/editor2/views/editor.html

```diff
@@ -6,12 +6,14 @@
 </a>
 <div ng-repeat="block in vm.blocks track by vm.generateBlockId(block)"
      class="block__container"
-     ng-class="{'block--selected': vm.blockToMoveIndex === $index}">
+     ng-class="{'block--selected': vm.blockToMoveIndex === $index,
+                'block--not-alone-and-empty': vm.blocks.length > 1 && (!block.body || block.body === '')}">
```

Comment:
` !block.body` is enough.

Here should be `vm.blocks.length > 1 && !block.body` ;)